update homepage to sync with github repo

adam3q opened the pull request:
1. Add Quickstart and FAQ button.
2. Improve the description of the Design section.
3. Replace the link to the Dragonfly section with the Helm doc.
4. Replace outdated remote graph driver with Docker/Moby docs.

imeoer commented on components/Design.vue:
```diff
@@ -47,11 +47,11 @@
-              <span class="inline-highlight">Kata Container</span>.
+              Supports container runtimes such as runC and 
+              <span class="inline-highlight">Kata</span>,
+              and provides Confidential Containers and vulnerability scanning capabilities
```
`provides Confidential Containers` The statement seems a little strange.

adam3q replied:
I changed it to `Supports container runtimes such as runC, Kata and confidential containers`

imeoer commented on components/Design.vue:
```diff
@@ -102,10 +102,10 @@
-              File metadata and blob data can be both validated in transmission
-              and at runtime.
+              In-Kernel EROFS or FUSE filesystems together 
+              with overlayfs provide full POSIX compatibility.
```
How about `In-kernel EROFS, FUSE, and VirtioFS support provides full POSIX compatibility.`

adam3q replied:
Great.

imeoer commented on components/Design.vue:
```diff
@@ -116,8 +116,8 @@
-              Support with Registry, OSS, NAS, Shared Disk, and P2P service,
-              compatible with the OCI spec.
+              Support with Registry, OSS, NAS, Shared Disk, and P2P service.
+              Compatible with the OCI images, and provide native eStargz images support.
```
We can't promise the estargz support now.

adam3q replied:
OK, I removed it.

adam3q committed:
update homepage to sync with github repo

1. Add Quickstart and FAQ button.
2. Improve the description of the Design section.
3. Replace the link to the Dragonfly section with the Helm doc.
4. Replace outdated remote graph driver with Docker/Moby docs.

Signed-off-by: Qinqi Qu <quqinqi@linux.alibaba.com>
Signed-off-by: Yan Song <imeoer@linux.alibaba.com>
